### DISCUSSION TITLE
Remove UUID directories upon downloading files

### DISCUSSION THREAD

**Discussion by igolan179** [iterate-ch/cyberduck#15285] [Q&A]
Since 8.5.5, Cyberduck started to create directories upon downloading files to edit with VScode:
![image](https://github.com/iterate-ch/cyberduck/assets/57070100/1147c5c0-94a7-4f9c-b2b2-14dbccc6f7d7)

I use a SCSS structure that needs the paths of the scss files to be without those folders, therefore I can't compile my main scss file.
Is there an option to disable those directories creation? 8.5.4 works perfect and I still use it for the above issue

**Comment by dkocher** [comment]
This has changed with the fix for #14114 in [^1]. There is currently  no workaround.

[^1]: #14116
